frappe/docs/user/en/tutorial/start.md

```diff
@@ -14,7 +14,7 @@ To start the development server, run `bench start`
 
 You can now open your browser and go to `http://localhost:8000`. You should see this login page if all goes well:
 
-<img class="screenshot" alt="Login Screen" src="{{docs_base_url}}/assets/img/login.png">
+<img class="screenshot" alt="Login Screen" src="http://i.imgur.com/yif2Uf3.png">
```

Comment:
@ratskin,

can you add the updated images in /frappe-bench/apps/frappe/frappe/docs/assets/img location instead of imgur.com?

Also please use the cropped images instead of full-screen images